ddtrace/contrib/trace_utils.py

```diff
@@ -415,3 +416,33 @@ def set_flattened_tags(
     for prefix, value in items:
         for tag, v in _flatten(value, sep, prefix, exclude_policy):
             span.set_tag(tag, processor(v) if processor is not None else v)
+
+
+def set_user(tracer, user_id, name=None, email=None, scope=None, role=None, session_id=None):
```

Comment:
Looking at the [documentation PR](https://github.com/DataDog/documentation/pull/14941/files) it seems like `set_user` should be a method in the tracer class. Do we plan to add this in a follow up PR?

Reply:
The first commits `set_user` was a method but I changed it by @Kyle-Verhoog recommendation, I'm waiting to close this PR to update the documentation :+1:

Reply:
Have we considered making this a top-level method?

e.g. `import ddtrace; ddtrace.set_user()` ?